nomad/job_endpoint.go

```diff
@@ -2391,3 +2398,19 @@ func (j *Job) GetServiceRegistrations(
 		},
 	})
 }
+
+func (j *Job) TagVersion(args *structs.JobApplyTagRequest, reply *structs.JobTagResponse) error {
```

Comment:
I think you just haven't implemented it yet, but don't forget you've still got to implement auth, metrics, and RPC forwarding at the top of this handler.